applications/worker/values.yaml

```diff
@@ -92,6 +101,8 @@ cloudsql:
 
 nodeSelector: {}
 
+tolerations: []
```

Comment:
I think we still want to set the default here to:

```yaml
- key: "removable"
   operator: "Equal"
   value: "true"
   effect: "NoSchedule"
```

Otherwise we'd be creating a breaking change for older architectures where the taint `removable: true` was set by default.